src/runtime/importhook.cs

```diff
@@ -39,11 +39,25 @@ internal static void Initialize()
             IntPtr mod = Runtime.IsPython3
                 ? Runtime.PyImport_ImportModule("builtins")
                 : Runtime.PyDict_GetItemString(dict, "__builtin__");
+            if (mod == IntPtr.Zero)
+            {
+                throw new PythonException();
+            }
```

Comment:
Has this ever happened for you? In any case, the call should be changed to `PyEval_GetBuiltins()`, while you're at it.

Reply:
`PyEval_GetBuiltins` don't have __import__ certainly
![default](https://user-images.githubusercontent.com/3406561/45144994-2c67c400-b1f2-11e8-8690-75347f10ccd9.png)

Reply:
That's odd, then `PyEval_GetBuiltins()` doesn't return `__builtin__` in Python 2. Sorry for the noise.